src/libs/actions/Policy.js

```diff
@@ -46,7 +48,9 @@ function getSimplifiedEmployeeList(employeeList) {
         .unique()
         .value();
 
-    return employeeListEmails;
+    const result = {};
+    _.each(employeeListEmails, email => result[email] = defaultEmployeeListEntry());
```

Comment:
Now that getSimplifiedEmployeeList is only used for memberList we can instead update the logic to be better. So instead of two blocks of logic doing different things we can simplify it such that its just one block doing exactly what we want. Something i had in mind is,
```js
function getSimplifiedEmployeeList(employeeList) {
    const employeeListEmails = _.chain(test)
                                .pluck('email')
                                .unique()
                                .reduce((acc, value) => { return {...acc, [value]: {}}}, 0)
                                .value();
    return employeeListEmails;
}
```